genienlp/train.py

```diff
@@ -520,7 +519,7 @@ def main(args):
 
     if hasattr(args, 'tensorboard') and args.tensorboard:
         logger.info(f'Initializing Writer')
-        writer = SummaryWriter(log_dir=args.tensorboard_dir, purge_step=start_iteration)
+        writer = SummaryWriter(log_dir=args.tensorboard_dir, purge_step=start_iteration, flush_secs=60)
```

Comment:
Btw, did we ever figure out why start iteration doesn't work to erase previous failed runs?

Reply:
What do you mean? The current behavior when `--resume` is set is that tensorboard will continue from the last iteration (or best iteration if that is the checkpoint we are resuming from). The original run and the resumed one will look like this:
![image](https://user-images.githubusercontent.com/60150701/103591624-87368680-4ea5-11eb-91ea-d95fa96dfa06.png)

Reply:
I mean when --resume is not set. In theory, passing purge_step=1 should remove all values with global_step < 1 in the same run (same tensorboard folder). In practice, the graph jumps.

Reply:
Possibly related to this fix in tensorboardX: https://github.com/lanpa/tensorboardX/pull/563

Reply:
Yeah, I updated tensorboardX version and it fixed this.